tests/output.rs

```diff
@@ -346,7 +368,7 @@ fn test_invalid_query_inline() {
         "rust_project",
         r#"
             $ tree-sitter-grep --query-source '(function_itemz) @function_item' --language rust
-            error: invalid query
+            error: couldn't parse query for Rust
```

Comment:
Does tree-sitter give any meaningful error messages when trying to parse a query that could be shown here?

Reply:
Ok ya that's a good idea, I made `CachedQueries`'s `HashMap` contain a `OnceLock<Result<...>>` instead of a `OnceLock<Option<...>>` but then its `.get_and_cache_query_for_language()` "API" still returns an `Option<...>` (ie it captures the query-parsing error but keeps that encapsulated)

And so then when it kicks in this "no successfully parsed queries behavior", if it only tried parsing for one language then it shows the error message from `tree-sitter` (figured that showing n query-parsing error messages would probably be "a lot" in the tried-to-parse-as-more-than-one-language case?)